src/mantarray_file_manager/exceptions.py

```diff
@@ -30,8 +30,11 @@ class FileAttributeNotFoundError(Exception):
     def __init__(
         self, attr_name: str, file_version: str, file_path: str,
     ):
-        if is_uuid(attr_name):
-            attr_name = METADATA_UUID_DESCRIPTIONS[UUID(attr_name)]
+        attr_description = (
+            f"{attr_name}, ({METADATA_UUID_DESCRIPTIONS[UUID(attr_name)]})"
```

Comment:
if the uuid is not in `METADATA_UUID_DESCRIPTIONS`, won't this raise a KeyError in the middle of trying to raise this error?  In general, you want to be very careful about logic that you put into an exception, to make sure it doesn't raise an error of its own

I think you'll need a test for a UUID that isn't in `METADATA_UUID_DESCRIPTIONS`, and then probably have a catch for a KeyError and then fall back to just the UUID itself.